pkg/server/httpAPI.go

```diff
@@ -439,7 +439,7 @@ func (h *HTTPHandler) serveConfig(rw http.ResponseWriter, req *http.Request) {
 	config["name"] = podName
 
 	initAdPeerURL := config["initial-advertise-peer-urls"]
-	protocol, svcName, namespace, peerPort, err := parsePeerURL(fmt.Sprint(initAdPeerURL))
+	protocol, svcName, namespace, peerPort, err := miscellaneous.ParseAdvertiseURL(fmt.Sprint(initAdPeerURL))
```

Comment:
Can this be replaced with a call to `CreateMemberPeerURL` instead? The logic here after calling `ParseAdvertiseURL` is identical to that in `CreateMemberPeerURL`.

Could you also add a simple comment before the function call like:
```go
// validate the URLs in the configuration file
```

Reply:
Done